### DISCUSSION TITLE
Welcome to APCEMM Discussions!

### DISCUSSION THREAD

**Discussion by sdeastham** [MIT-LAE/APCEMM#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Calebsakhtar** [comment]
Hello! I am Caleb and I am a PhD student working on contrail persistence and avoidance at the University of Cambridge. I am currently studying APCEMM as a part of my PhD. I intend to share any findings and help develop APCEMM as appropriate.

I would like to extend a great thank you to the MIT Team and especially @sdeastham and @michaelxu3 for all of the work done in this repo.
